src/http/request.ts

```diff
@@ -125,6 +131,20 @@ export abstract class Request<S = undefined> {
     };
   }
 
+  /**
+   * Send a post request using the Navigator.sendBeacon api.
+   * - Note that only url parameters are supported.
+   * - The sendBeacon api does not return any response.
+   */
+  public sendBeacon<EK extends LkErrorKey = never>(rawFetchConfig: SendBeaconConfig<EK>): void {
```

Comment:
Consider Returning a promise, so that the return value of `releaseEditLock()` is always a promise:

```suggestion
  public async sendBeacon<EK extends LkErrorKey = never>(rawFetchConfig: SendBeaconConfig<EK>): Promise<void> {
```

Reply:
Makes sense 👍